conference/src/main/java/com/kyriba/conference/management/service/HallServiceImpl.java

```diff
@@ -24,7 +29,7 @@
 
 
   @Override
-  public HallResponse findHall(long id)
+  public HallResponse findHall(@Valid @Positive long id)
```

Comment:
There is no reason to duplicate in interface and implementation.

Reply:
The same comment is valid for other methods as well.

Reply:
so where to put these annotations? interface or implementation?

Reply:
it is required in interface